.vib/appsmith/cypress/cypress/integration/appsmith_spec.js

```diff
@@ -0,0 +1,43 @@
+/// <reference types="cypress" />
+import {
+  random,
+} from '../support/utils';
+
+it('allows to log in and out', () => {
+  cy.login();
+  cy.get('header div[class*="profile-menu"]').click();
+  cy.contains('Sign Out').click();
+  cy.contains('sign in');
+});
+
+it('allows to create a new project', () => {
```

Comment:
Maybe I am wrong because I don't have the whole background on the app, but I think would additionally check that the project can be **deployed** (and that it is accessible) as I consider it to be within the "core" goal of this asset.

Reply:
Didn't add it because it opens a new tab and Cypress stops working

Reply:
I have found a workaround for this. It seems there is an `a` element with the URL to the deployed app with `target="_blank"`. You can find it with the selector `[class*="current-deployed"]`. From there you can obtain the value of `href` or remove the `target` element.

If you want, you can also test whether the app is accessible from the outside making it public and then visiting the URL after login out.

Reply:
Added deploy test. About the logout, I don't see much value there because that would be an issue in Appsmith, not on the packaging. We already checked that the deploy URL is accessible so nginx is properly configured.